### ISSUE TITLE
[BUG] Flyte logger overflooding when logging inputs to function, can't avoid that

### ISSUE THREAD

**Issue by jiwidi** [flyteorg/flyte#4695]
### Describe the bug

So I'm sharing a fairly big dictionary between flyte task and its being printed everytime when the logger logs

" {"asctime": "2024-01-09 10:13:27,582", "name": "flytekit", "levelname": "INFO", "message": "Invoking main.function_name with inputs: {DICT}"

I can not change this and is a pain in the ass. These messages are set to info level and not debug as it leaves no room for other logging level to be use before warning. If these were debug level logs i could just set the logger to info avoid them.

 I just want some simple logs that tell me on what step the pipeline is running at the moment without the need of seeing each task inputs.

### Expected behavior

 I just want some simple logs that tell me on what step the pipeline is running at the moment without the need of seeing each task inputs.

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by kumare3** [comment]
@jiwidi i am indeed reviewing all the logs and working on improving them.
would you be open to contributing.

**Comment by kumare3** [comment]
fix log-levels

**Comment by jiwidi** [comment]
@kumare3 Yes I would be open to contributing. Have never worked with flyte so will need some guide to what code and files i would need to edit, if you can provide me i can help out

**Comment by jiwidi** [comment]
@kumare3 hello?
